features/support/cucumber_rails_helper.rb

```diff
@@ -1,25 +1,36 @@
 # frozen_string_literal: true
 
+require 'rails'
+require 'cucumber'
+require 'capybara'
+
 module CucumberRailsHelper
   def rails_new(options = {})
     validate_rails_new_success(run_rails_new_command(options))
-    clear_bundle_env_vars(options[:name])
+    cd options[:name]
+    configure_rails_gems
+    configure_rails_requires
+    configure_rails_layout
+    clear_bundle_env_vars
   end
 
   def install_cucumber_rails(*options)
     add_conditional_gems(options)
 
-    add_gem 'capybara', group: :test
+    add_gem 'cucumber', Cucumber::VERSION, group: :test
+    add_gem 'capybara', Capybara::VERSION, group: :test
```

Comment:
Use the version of `cucumber` and `capybara` as locked in the Appraisal gem file.